### DISCUSSION TITLE
Missing master.tar.gz

### DISCUSSION THREAD

**Discussion by netidjen** [swagger-api/swagger-ui#6890] [General]
I'm using v1.2.1 swagger ui that only compatible with v1.5.22 swagger-jaxrs. Yes, it's for java ee project. Did someone delete the archive folder? 
https://github.com/swagger-api/swagger-ui/archive/master.tar.gz

**Comment by hkosova** [comment]
Hi @netidjen, the archive link that you posted works for me.
